app/src/main/res/menu/menu_bottom_nav.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:id="@+id/directions"
+        android:icon="@drawable/ic_directions"
+        android:title="Directions" />
```

Comment:
Move all of these hardcoded strings to string resources

Reply:
Done!